File: pc/plugins/round_number.py
import re
from core.plugin import Plugin


class RoundNumberPlugin(Plugin):
    def translate(self, line):
        match = re.fullmatch(r'四捨五入\s+(.+)', line)

        if not match:
            return None

        value = match.group(1)

        return f"print(round({value}))"

File: web/plugins/round_number.js
import { WebPlugin } from "../engine/plugin.js";

export class RoundNumberPlugin extends WebPlugin {
    constructor() {
        super("四捨五入");
    }

    async execute(line, context) {
        const expression = line
            .replace(/^四捨五入\s*/, "");

        const value = Number(
            context.parser.parseValue(
                expression,
                context.runtime
            )
        );

        context.runtime.write(
            Math.round(value)
        );
    }
}